Gemfile

```diff
@@ -7,8 +7,11 @@ gem "propshaft"
 # Use pg as the database for Active Record
 gem "sqlite3"
 gem "sqlite-vec"
+gem "neighbor"
+gem "tokenizers"
```

Comment:
Pre-defined tokenizers that we can use with a tokenizer parameter file, we use them to build a tokenizer for XLM Roberta CLIP